components/sections/Header/HeaderDesktop/HeaderDeskop.tsx

```diff
@@ -0,0 +1,68 @@
+// React/Relay
+import React, { FC } from "react";
+
+// Component
+import { Props } from "./HeaderDesktop.types";
+import styles from "./HeaderDesktop.scss";
+
+export const Header: FC<Props> = (props) => {
+  return (
+    <section className={styles.container}>
+      <div className={styles.socialMedia}>
+        {props.social_media.map((socialMedia, index) => {
+          return (
+            <a
+              className={styles.socialIcon}
+              key={index}
+              href={
+                socialMedia.social_link && (socialMedia.social_link as any).url
+              }
+            >
+              <img
+                src={
+                  socialMedia.social_icon &&
+                  (socialMedia.social_icon as any).url
+                }
+              />
+            </a>
+          );
+        })}
+      </div>
+      <div className={styles.headerContainer}>
+        <img
+          className={styles.icon}
+          src={(props.pata_pila_icon as any).url}
+          alt="Pata Pila"
+        />
+        <div className={styles.linksContainer}>
+          {props.header_links.map((link, index) => {
+            return (
+              <a
+                key={index}
+                href={link.link_url && (link.link_url as any).url}
+                className={styles.link}
+              >
+                {link.link_title[0].text}
+              </a>
+            );
+          })}
+          {props.action_buttons.map((button, index) => {
+            return (
+              <button
+                key={index}
+                className={styles.actionButton}
+                style={{
+                  backgroundColor: button.background_color ?? "transparent",
+                  borderColor: button.border_color,
```

Comment:
What would happen if the user doesn't define these properties?

Reply:
i will put default values